scan/lib/scan/test_command_generator.rb

```diff
@@ -69,6 +69,14 @@ def options # rubocop:disable Metrics/PerceivedComplexity
       options << "-enableAddressSanitizer #{config[:address_sanitizer] ? 'YES' : 'NO'}" unless config[:address_sanitizer].nil?
       options << "-enableThreadSanitizer #{config[:thread_sanitizer] ? 'YES' : 'NO'}" unless config[:thread_sanitizer].nil?
       if FastlaneCore::Helper.xcode_at_least?(11)
+        if config[:cloned_source_packages_path] && !options.include?("-clonedSourcePackagesDirPath #{config[:cloned_source_packages_path].shellescape}")
+          options << "-clonedSourcePackagesDirPath #{config[:cloned_source_packages_path].shellescape}"
+        end
+
+        if config[:package_cache_path] && !options.include?("-packageCachePath #{config[:package_cache_path].shellescape}")
+          options << "-packageCachePath #{config[:package_cache_path].shellescape}"
+        end
```

Comment:
This as well as the other changes in the MR adds the ability to pass: `packageCachePath` to `xcodebuild`

I've noticed performance improvements (less networking) when providing both `packageCachePath` and `clonedSourcePackagesDirPath` - they do end up with sort of similar contents.

From `xcodebuild --help`:
| Argument | Description |
| - | - |
| `-packageCachePath` | path of caches used for package support |
| `-clonedSourcePackagesDirPath PATH` | specifies the directory to which remote source packages are fetch or expected to be found` |